### ISSUE TITLE
Walmart container problem

### ISSUE THREAD

**Issue by MSDNDylan** [jef/streetmerchant#1106]
### Description
Whenever the program calls the lookupCard() function within lookup.ts it isn't getting back the response object after it is calling page.goto(link.url)

I have been testing this with the walmart store only. When I set headless to false I see that it does go to the page in chromuim

**Comment by MSDNDylan** [comment]
To give some more detail basically I was getting "OUT OF STOCK" on an item that I know is in stock I swapped out a link in the walmart store model. When I enabled debug mode it still says the product is not in stock but it also stated 'NO RESPONSE'. Did some further digging and narrowed it down to the lookupCard() function in lookup.ts is where it is printing and report not in stock I created my own message there and same response. I am not sure of the response or why it is not returning anything into the response const but it just isn't. I turned headless to false and it opened the browser and behaved normally visually its just not getting back the page in the code for some reason. I can confirm its going to the correct product that is in stock.
